measure-u1.cc

```diff
@@ -90,6 +90,15 @@ int main(int ac, char* av[]) {
       os.fill(prevf);
       os << ".dat" << std::ends;
       compute_spacial_loops(U, os.str());
+      for(size_t xp=0;xp<3; xp+=1){
```

Comment:
advice: maybe here you want to generalize to the number of dimensions ndims? (instead of 3)

Reply:
That is part of @urbach's code, I did not touch it

Reply:
Ok, I guess it is the relic of some test. @urbach do you agree that the limits should be the lattice spatial extents? 
If I understood correctly the ```compute_spacial_loops()``` function, the 3 arguments are the components of the position vector connecting the two quarks, and ```compute_spacial_loops()``` averages over all possible permutations of the 3 spatial directions.